realm/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -326,6 +334,11 @@ public Builder deleteRealmIfMigrationNeeded() {
             return this;
         }
 
+        public Builder durability(SharedGroup.Durability durability) {
```

Comment:
We cannot expose SharedGroup to the public facing API. Either we move the enum, or we wrap it, or we should use some other mechanism. @cmelchior @nhachicha @kneth what do you guys think?